File: php/sample.php
<?php

  $URL = '<url>';
  $TOKEN = '<access_token>';
  $AUDIO = '<audio_file_path>';

  $ch = curl_init();
  curl_setopt_array($ch, [
    CURLOPT_URL => $URL,
    CURLOPT_POST => true,
    CURLOPT_POSTFIELDS => [
      'file' => new CURLFile($AUDIO),
    ],
    CURLOPT_RETURNTRANSFER => true,
    CURLOPT_HTTPHEADER => [
      'Content-type: multipart/form-data',
      "Authorization: Bearer $TOKEN"
    ]
  ]);

  $res = curl_exec($ch);
  curl_close($ch);

  $res = json_decode($res, true);

  if ($res['code'] == 200) {
    foreach ($res['results'] as $d) {
      foreach (array_keys($d) as $key){
        print($key . ':'. $d[$key] . "\n");
      }
    }
  } else {
    print($res['code']);
    print($res['error']['message']);
  }

File: python/sample.py
import requests

URL = '<url>'
TOKEN = '<access_token>'
AUDIO = '<audio_file_path>'

def main():
    headers = {'Authorization':f'Bearer {TOKEN}'}
    audio_file = {'file':open(AUDIO,'rb')}

    res = requests.post(URL, files=audio_file, headers=headers).json()
    if res['code'] == 200:

        for data in res['results']:
            for key in data.keys():
                print(f'{key}:', data[key])
    else:
        print(res['code'])
        print(res['error']['message'])

if __name__ == '__main__':
    main()
